### PR TITLE
replace tags with tagWithEntityId

### PR DESCRIPTION
**Pull request by Kelerchian**

### CODE REVIEW

File: machine-check/examples/example.test.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { describe, expect, it } from '@jest/globals'
-import { Door, Control, HangarBay } from './example-proto.js'
+import { Door, Control } from './example-proto'
```
Comment (rkuhn) [inline machine-check/examples/example.test.ts:2]: I thought that ESM requires the `.js` suffix?

File: machine-runner/src/design/protocol.ts [hunk 2]
```diff
@@ -38,7 +39,7 @@ export namespace SwarmProtocol {
   /**
    * Construct a SwarmProtocol
    * @param swarmName - The name of the swarm protocol
-   * @param tags - the tags used to mark the events passed to Actyx
+   * @param tagString - the tags used to mark the events passed to Actyx
```
Comment (rkuhn) [inline machine-runner/src/design/protocol.ts:42]:
```suggestion
   * @param tagString - the base tag used to mark the events passed to Actyx
```

**Review by rkuhn — APPROVED** [review]
Yes, that’s even better. Just two minor points.

File: machine-runner/src/design/protocol.ts (hunk 2 above)
Comment (Kelerchian) [reply machine-runner/src/design/protocol.ts:42]: Thanks. I missed out all the example jsdocs too.

File: machine-check/examples/example.test.ts (hunk 1 above)
Comment (Kelerchian) [reply machine-check/examples/example.test.ts:2]: You're right. I was tinkering with this and forgot to revert. I'm curious though how `tsc` perceive this compared to `.js`

**Pull request merged**